modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -100,11 +103,13 @@ export default {
   },
   mounted() {
     this.$refs.spacer.style.height = `${this.$refs.adminBar.offsetHeight}px`;
-    this.menuItems = this.items.map(item => {
+    const itemsSet = this.filterEmptySchema(klona(this.items));
+
+    this.menuItems = itemsSet.map(item => {
       if (item.items) {
         item.items.forEach(subitem => {
           // The context menu needs an `action` property to emit.
-          subitem.action = subitem.name;
+          subitem.action = subitem.action || subitem.name;
```

Comment:
we're actually setting an action property now.

Reply:
I'm OK with it if you want to Action All The Things.